doc/examples/effects/simple/lib/main_move.dart

```diff
@@ -1,4 +1,5 @@
 import 'package:flutter/material.dart';
+import 'package:flame/extensions/vector2.dart';
```

Comment:
local imports are relative, weir that this file has some many absolute imports.

Reply:
But this is the example, they never used relative imports I think? Since they aren't really part of the lib?

Reply:
Yeah, you are right! Sorry for the confusion